### PR TITLE
Bitnami now uses Notation for signing and verifying blog post

### LINKED ISSUE #394: Add Bitnami as a Notation adopter and a blog entry
As Notation is supported in DockerHub https://github.com/notaryproject/notaryproject.dev/pull/393 The Bitnami team is working on signing the containers and Helm charts in DockerHub using Notation. E.g. https://hub.docker.com/r/bitnami/postgresql/tags

This issue is to create a blog entry and add Bitnami as adopter of Notation as well.

### PR DESCRIPTION
**Pull request by beltran-rubo**
Bitnami container images using notation. Closes #394.

### CODE REVIEW

File: content/en/blog/2024/bitnami-using-notation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+In December 2023, [we announced](https://tanzu.vmware.com/content/tanzu-application-catalog-resources/tanzu-application-catalog-leverages-notation) that Tanzu Application Catalog, the enterprise edition of Bitnami Application Catalog, started making use of Notation as a tool for signing and verifying open container initiative (OCI) artifacts (e.g. container images, Helm charts, and metadata bundles.
+
+Now, we are happy to have extended our collaboration with Notation and announce the extension of this capability to the community edition of Bitnami-packaged container images and Helm charts in DockerHub as well.
+
+To know more about the benefits that the Bitnami users stand to enjoy with this integration and to learn how to verify the signature of a Bitnami-package, check out [this blog](https://blog.bitnami.com/2024/03/bitnami-packaged-containers-and-helm.html).
```
Comment (FeynmanZhou) [inline content/en/blog/2024/bitnami-using-notation.md:16]: We will hold on merging this PR until this link is live: https://blog.bitnami.com/2024/03/bitnami-packaged-containers-and-helm.html

File: content/en/blog/2024/bitnami-using-notation.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Bitnami now uses Notation for signing and verifying images on DockerHub
```
Comment (FeynmanZhou) [inline content/en/blog/2024/bitnami-using-notation.md:2]:
```suggestion
title: Bitnami now uses Notation for signing and verifying images and Helm Charts on Docker Hub
```

Comment (FeynmanZhou) [reply content/en/blog/2024/bitnami-using-notation.md:2]: To align this title with https://github.com/notaryproject/notaryproject.dev/pull/396/files#diff-40a9baefb9102a034f95dbfda47c8038ba64e5c4834ad6d0224f385ecba42d0dR14

**Review by yizha1 — APPROVED** [review]
LGTM

Thanks @beltran-rubo

File: content/en/blog/2024/bitnami-using-notation.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Bitnami now uses Notation for signing and verifying images and Helm Charts on Docker Hub
```
Comment (beltran-rubo) [inline content/en/blog/2024/bitnami-using-notation.md:2]:
```suggestion
title: Bitnami now uses Notation for signing and verifying containers and Helm charts on Docker Hub
```

File: content/en/blog/2024/bitnami-using-notation.md (hunk 1 above)
Comment (beltran-rubo) [reply content/en/blog/2024/bitnami-using-notation.md:16]: Correct

**Review by priteshbandi — DISMISSED** [review]
LGTM but please dont merge the PR until the link to [blog](https://blog.bitnami.com/2024/03/bitnami-packaged-containers-and-helm.html) is live

**Review by FeynmanZhou — APPROVED** [review]
LGTM
Thanks @beltran-rubo

File: content/en/blog/2024/bitnami-using-notation.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: Bitnami now uses Notation for signing and verifying containers and Helm charts on Docker Hub
+author:  Beltran Rueda
+date: 2024-03-18
+draft: true
```
Comment (beltran-rubo) [inline content/en/blog/2024/bitnami-using-notation.md:5]:
```suggestion
draft: false
```

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Comment by yizha1** [comment]
Merged this PR, as Bitnami blog was published https://blog.bitnami.com/2024/03/bitnami-packaged-containers-and-helm.html

**Pull request merged**